src/main.ts

```diff
@@ -108,6 +110,17 @@ export async function finish(): Promise<void> {
 
     //TODO: end tracing and collect paths in filelist.txt
 
+    //TODO: Until we have tracing wired up, the used files are hard-coded.
+    if (fs.existsSync('main.rb')) {
+      cachePaths.push('main.rb')
+    }
+    if (fs.existsSync('Gemfile')) {
+      cachePaths.push('Gemfile')
+    }
+    if (fs.existsSync('Gemfile.lock')) {
+      cachePaths.push('Gemfile.lock')
```

Comment:
This would write the contents of those files into the cache, we don't really need that, right?